src/fides/api/app_setup.py

```diff
@@ -162,6 +166,11 @@ async def run_database_startup() -> None:
         return
     finally:
         db.close()
+
+    if not CONFIG.test_mode:
+        # Default notices subject to change, so preventing these from
+        # loading in test mode to avoid interfering with unit tests.
+        load_default_privacy_notices()
```

Comment:
We have hundreds of existing tests across privacy notices, experiences, and experience configs whose notices would conflict with those that we're loading on startup.  Especially because we're still figuring out "regions" - rather than loading these in tests by default and changing all of our tests to accommodate this, I'm just not loading them in test mode.

I do call this method separately in a unit test that will fail if any validation fails as a sanity check.